packages/host/tests/acceptance/interact-submode-test.gts

```diff
@@ -795,6 +795,30 @@ module('Acceptance | interact submode tests', function (hooks) {
       );
       await click('[data-test-stack-card-index="1"] [data-test-edit-button]');
     });
+
+    test<TestContextWithSave>('new card is created in the current realm', async function (assert) {
+      assert.expect(1);
+      await visitOperatorMode({
+        stacks: [
+          [
+            {
+              id: `${testRealmURL}Person/fadhlan`,
+              format: 'edit',
+            },
+          ],
+        ],
+      });
+      this.onSave((url) => {
+        if (url.href.includes('Pet')) {
+          assert.ok(
+            url.href.startsWith(testRealmURL),
+            `The pet card is saved in the current realm ${testRealmURL}`,
+          );
```

Comment:
for this test module, the default writable realm is actually `testRealm2URL` since it is first lexically